config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show] do
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] We should have routes that look like this:

  ```
  users/user_id/posts/:id    # post show
  users/user_id/posts         # post index
  ```

  That means we need to nest these routes to achieve that.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

Reply:
Done 

<img width="765" alt="image" src="https://user-images.githubusercontent.com/43815180/205929702-ae32a6f8-a618-4b24-aa38-32d9cec48d77.png">